### ISSUE TITLE
Add argparse option for NWL mode

### ISSUE THREAD

**Issue by gonuke** [svalinn/parastell#63]
Add an option in argparse that allows parastell to run in an NWL mode that only generates as much as necessary to run a NWL calculation

**Comment by connoramoreno** [comment]
Should this be addressed in PR #65?

**Comment by gonuke** [comment]
I think there is some refactoring that might happen as described in #66

**Comment by gonuke** [comment]
I think this can be a separate PR from #65 as it's probably an addition to `parastell.py`

**Comment by gonuke** [comment]
This depends on #53

**Comment by gonuke** [comment]
Closing this as largely redundant to #53
